components/Inputs/TileSelectInput.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Text, TouchableOpacity } from 'react-native';
+import { FImage } from 'components/Composition/FImage';
+import placements from 'themes/placements';
+import sizes from 'themes/sizes';
+import colors from 'themes/colors';
+import fonts from 'themes/fonts';
+
+export const TileSelectInput = ({
+  iconSize, iconDefault, iconPressed, label, setValue, value,
+}) => {
+  const [
```

Comment:
ten stan jest do wywalenia, Jeżeli chcesz żeby zmieniały się style itd. to musisz to uzależnić po prostu od value. Jeżeli  ją ma to znaczy że ma się ustawić jako zaznaczony (dostać inne style). Czyli na onPress dajesz setValue => onPress = {setValue} a za,miast isPressed dajesz value